posttraining/launch_alpaca_eval.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#SBATCH --job-name=alpaca_eval  # Job name
+#SBATCH --nodes=1
+#SBATCH --partition=dev-g
+#SBATCH --time=01:00:00
+#SBATCH --ntasks-per-node=8
+#SBATCH --cpus-per-task=7
+#SBATCH --exclusive=user
+#SBATCH --hint=nomultithread
+#SBATCH --gpus-per-node=mi250:8
+#SBATCH --account=project_462000353
```

Comment:
would be good to paramertize this and other params and pass them downwards from from posttraining_evals.sh so it will be more portable, customisable